#修改wifi_user_config.rtw_tx_pwr_lmt_enable=1(0:禁用；1:使用用户配置；2:使用efuse配置)
#芯片支持40MHz频宽
#修改no beacon断开时长
#关闭自动重连
#修改AP模式连接上限
#关闭低功耗

diff --git a/component/soc/usrcfg/amebadplus/ameba_wificfg.c b/component/soc/usrcfg/amebadplus/ameba_wificfg.c
index 99701cba4..9da94b1ce 100644
--- a/component/soc/usrcfg/amebadplus/ameba_wificfg.c
+++ b/component/soc/usrcfg/amebadplus/ameba_wificfg.c
@@ -21,11 +21,11 @@ _WEAK void wifi_set_user_config(void)
 	/* below items for user config, for details, see wifi_user_conf in ameba_wificfg_common.h */
 	wifi_user_config.concurrent_enabled = 1;
 	wifi_user_config.softap_addr_offset_idx = 1;
-	wifi_user_config.fast_reconnect_en = 1;
-	wifi_user_config.auto_reconnect_en = 1;
+	wifi_user_config.fast_reconnect_en = 0;
+	wifi_user_config.auto_reconnect_en = 0;
 	wifi_user_config.auto_reconnect_count = 10;
 	wifi_user_config.auto_reconnect_interval = 5;
-	wifi_user_config.no_beacon_disconnect_time = 9;  /* unit 2s, default 18s */
+	wifi_user_config.no_beacon_disconnect_time = 3;  /* unit 2s, default 18s */
 
 #if (defined(CONFIG_WHC_INTF_SDIO) || defined(CONFIG_WHC_INTF_USB)) || (defined(CONFIG_WHC_BRIDGE))
 	skb_num_np_rsvd = 10; /*4 for rx_ring_buffer + 2 for mgnt trx + 4 for spido rx_ring_buffer */
@@ -61,9 +61,9 @@ _WEAK void wifi_set_user_config(void)
 	/*Regulatory related*/
 	wifi_user_config.country_code[0] = 0;
 	wifi_user_config.country_code[1] = 0;
-	wifi_user_config.bw_40_enable = 0;
+	wifi_user_config.bw_40_enable = 1;
 	wifi_user_config.freq_band_support = RTW_SUPPORT_BAND_MAX;
-	wifi_user_config.tx_pwr_table_selection = 2;
+	wifi_user_config.tx_pwr_table_selection = 1;
 	wifi_user_config.rtw_802_11d_en = 0;
 	wifi_user_config.rtw_trp_tis_cert_en = RTW_TRP_TIS_DISABLE;
 	wifi_user_config.rtw_edcca_mode = RTW_EDCCA_NORM;
@@ -71,18 +71,18 @@ _WEAK void wifi_set_user_config(void)
 	wifi_user_config.probe_hidden_ap_on_passive_ch = 1;
 
 	/* IPS(Inactive Power Save), power save when wifi unconnected */
-	wifi_user_config.ips_enable = 1;
+	wifi_user_config.ips_enable = 0;
 	wifi_user_config.ips_level = RTW_IPS_WIFI_OFF;
 	wifi_user_config.ips_ctrl_by_usr = 0;
 
 	/* LPS(Legacy Power Save), power save when wifi connected */
-	wifi_user_config.lps_enable = 1;
+	wifi_user_config.lps_enable = 0;
 	wifi_user_config.lps_listen_interval = 0; // lps_listen_interval
 	wifi_user_config.lps_rx_unicast_pkt_timeout = 40;	/* set rx unicast packet timeout in LPS, unit:ms, max_value:100*/
 	wifi_user_config.wowlan_rx_bcmc_dis = 0;
 
 	/* Softap related */
-	wifi_user_config.ap_sta_num = 5;	/*should not exceed 14 */
+	wifi_user_config.ap_sta_num = 14;	/*should not exceed 14 */
 	wifi_user_config.ap_polling_sta = 0;
 
 	/* MISC */
